step_impl/step_impl.py

```diff
@@ -23,21 +24,64 @@ def random_amount():
     return random.randint(1, 100000) / 100
 
 
+def random_tran_id():
+    return random.randint(1, 100000000)
+
+
+def random_currency():
+    list_currency = ["RON", "EUR", "USD"]
+    return random.choice(list_currency)
+
+
+def random_debit_credit():
+    list_d_c = ["D", "C"]
+    return random.choice(list_d_c)
+
+
+def random_tran_status():
+    list_statuses = ["Pending", "Processed"]
+    return random.choice(list_statuses)
+
+
+def random_iban():
+    list_ibans = [
+        "RO30BRDE450SV22222200002",
+        "RO73BRDE450SV22222200004",
+        "RO44BRDE450CR22222200010",
+    ]
+    return random.choice(list_ibans)
+
+
 @step("Add a transaction every <frequency> seconds for <duration> seconds")
 def add_transactions(frequency, duration):
     print(
         f"Adding a transaction every {frequency} seconds for {duration} seconds . . ."
     )
-    sql = """INSERT INTO transactions(amount, time)
-             VALUES(%s, now()) RETURNING id;"""
+
+    # sql = """INSERT INTO transactions(amount, time)
```

Comment:
This code [should be deleted, not commented out](https://blog.submain.com/delete-commented-code-without-reading/).